erpnext/accounts/utils.py

```diff
@@ -1094,7 +1094,7 @@ def get_companies():
 def get_children(doctype, parent, company, is_root=False):
 	from erpnext.accounts.report.financial_statements import sort_accounts
 
-	parent_fieldname = "parent_" + doctype.lower().replace(" ", "_")
```

Comment:
First of all, thanks for reviewing.

@ankush Not sure whether you ask me. Although not necessary (since binding to this specific class ) I still changed it, in order to follow best practise and be consistent with the general frappe framework approach. And I think there was a good reason for adding the frappe.scrub function. Otherwise you could even hardcode the name here. By fixing it you also prevent copy&paste errors by newby developers.